deps/chakrashim/src/v8booleanobject.cc

```diff
@@ -23,10 +23,12 @@
 namespace v8 {
 
 using jsrt::ContextShim;
+using jsrt::IsolateShim;
 
-Local<Value> BooleanObject::New(bool value) {
```

Comment:
Yeah, i was debating that myself if i should keep it or remove it. AFAIK, no one can use the deprecated because it will fail the build of that module because of `V8_DEPRECATED` macro, right? But checking v8's code, they have kept it so would be ok to just leave it instead of deleting.